extensions/functions_comparison.yaml

```
@@ -62,6 +62,9 @@ scalar_functions:
     description: Whether a value is null.
     impls:
       - args:
+          - name: nan_is_null
+            options: [ TRUE, FALSE ]
+            required: false
           - value: any1
         return: BOOLEAN
         nullability: DECLARED_OUTPUT
```

It doesn't really make sense to apply this function to types that cannot be null, so
```suggestion
          - value: any1?
        return: BOOLEAN
        nullability: DISCRETE
```